Cargo.toml

```diff
@@ -22,6 +22,7 @@ members = [
 ]
 exclude = []
 default-members = [
+    "cts_runner",
```

Comment:
This was needed or else:
```
cargo run --manifest-path ../Cargo.toml --bin cts_runner -- ./tools/run_deno 'webgpu:api,validation,createBindGroup:binding_resources,device_mismatch:*'
error: no bin target named `cts_runner`.
Available bin targets:
    naga
    spv_parser
    wgsl_parser
    glsl_parser
    ir
    play
    wgpu-examples
    wgpu-info
```
or alternative is to run with `cargo run --manifest-path ../cts_runner/Cargo.toml -- ./tools/ru
n_deno 'webgpu:api,validation,createBindGroup:binding_resources,device_mismatch:*'`

Reply:
I think this is because cts_runner requires v8_rusty that does not have artifacts for aarch64 (ios ans android) hence the build were failing. I just updated line in readme to use `-p cts_runner`.

Although that's to long to write; justfile or xtask should be used for this.